container-disc/src/main/java/com/yahoo/container/jdisc/ConfiguredApplication.java

```diff
@@ -235,6 +239,11 @@ private void watchPortChange() {
         }
     }
 
+    void recofigure(QrConfig qrConfig) {
```

Comment:
Typo in method name

Reply:
Thanks, fixed in #19293

Reply:
Have to redo this one. The first part should not be there.